translator/lib/SPIRV/SPIRVInternal.h

```diff
@@ -1183,11 +1183,13 @@ union ShaderInOutMetadata {
                                       // output (tessellation shader)
     uint64_t StreamId           : 2;  // ID of output stream (geometry shader)
     uint64_t IsXfb              : 1;  // Whether this is for transform feedback
+    uint64_t IsBlockArray       : 1;  // Whether we are handling block array
```

Comment:
I am afraid this change is conflict with my shadermodule change, I reorder the field of this struct, please resolve it after my change is merged.

Reply:
I will still refine the field name because XfbLoc is unclear for code reader,